spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java

```diff
@@ -16,35 +16,31 @@
 
 package org.springframework.integration.handler;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```

Comment:
Right, and Checkstyle says you about that:
```
:spring-integration-core:checkstyleTest[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java:19: Using the '.*' form of import should be avoided - org.junit.Assert.*. [AvoidStarImport]

[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java:20: Using the '.*' form of import should be avoided - org.mockito.Mockito.*. [AvoidStarImport]
```
For this purpose you can run testing like `gradlew clean check --parallel`. 
Sometime if I need really only checkstyle, I add `-x test` flag as well.

I'm waiting for your fix and merge...